app-web/src/constants/ariaLabels.js

```diff
@@ -4,3 +4,6 @@ export const ARIA_LABEL_FILE_IN_REPO =
 export const ARIA_LABEL_RESOURCE = 'Learn more about this article';
 export const ARIA_LABEL_FILTER_SELECT = 'Select to filter cards by the resource type';
 export const ARIA_LABEL_TO_GITHUB_USER = 'Select to view this authors Github Profile';
+export const ARIA_LABEL_TO_GITHUB_ISSUES = "Select to view this respositories' issues";
```

Comment:
better to make descriptions more details by saying 'this Github repos....'

Reply:
changed!